subsys/mgmt/ec_host_cmd/Kconfig

```diff
@@ -25,6 +25,7 @@ config EC_HOST_CMD_HANDLER_TX_BUFFER
 	int "Buffer size in bytes for TX buffer shared by all EC host commands"
 	default 0 if EC_HOST_CMD_BACKEND_ESPI
 	default 0 if EC_HOST_CMD_BACKEND_SHI
+	default 256 if EC_HOST_CMD_BACKEND_UART
```

Comment:
this line isn't needed as it will use the final `default 256` line, unless multiple of these are enabled for a single build? If so, then the top most one with the first enabled Kconfig would be used, might want to re-order this and below Kconfig if that is a usecase

Reply:
I'm aware it isn't needed in this form, but I want to show UART default needs explicitly.